modules/single_cell/starsolo.jst

```diff
@@ -16,6 +16,7 @@
 {% set temp_dir %}temp/{{ sample.gltype }}/expression_estimates/starSOLO/{{ sample.name }}{% endset %}
 {% set task %}starsolo_count_{{ sample.name }}{% endset %}
 {% set json %}{{ alignment_outdir }}/stats/{{ sample.name }}.starSOLO.bam.Barcodes.stats.json{% endset %}
+{% set chemistry = constants.phoenix.scrna_chemistry_options[sample.assayCode] | default(constants.phoenix.scrna_chemistry_options['unknown']) %}
```

Comment:
For star solo I don't think we can do an unknown, the only way to configure correctly is to know